apps/src/javalab/PlaygroundVisualizationColumn.jsx

```diff
@@ -60,6 +64,15 @@ const styles = {
     width: 800,
     height: 800,
     overflow: 'hidden'
+  },
+  playgroundBackground: {
+    position: 'absolute',
+    width: 800,
```

Comment:
What's the distinction between the 400x400 size we use in Javabuilder vs. 800x800 here? Do we just scale between the two?

Reply:
Yeah, so I believe 800x800 is the maximum size the visualization can have and then we scale it based on available space. The 400x400 represents the pixel coordinate system that students use for setting x, y, width, height. Those coordinates and sizes are scaled to the actual size of the visualization. That scaling logic is here: https://github.com/code-dot-org/code-dot-org/blob/8e789eb2c579aba273fc4c5ef7bd75148791f7bd/apps/src/javalab/JavalabPanels.jsx#L149